go/teams/implicit.go

```diff
@@ -135,7 +147,7 @@ func lookupImplicitTeamAndConflicts(ctx context.Context, g *libkb.GlobalContext,
 	if err != nil {
 		return teamID, teamName, impTeamName, tlfID, conflicts, err
 	}
-	referenceImpName, err := FormatImplicitTeamDisplayName(ctx, g, impTeamNameWithoutConflict)
+	referenceImpName, err := FormatImplicitTeamDisplayName(ctx, g, impTeamName)
```

Comment:
The idea here was to compare the display names but ignoring the suffix part. As it stands I expect this comparison to fail when looking up a conflict. Because teamDisplayName will not have a suffix, though that is itching to change (https://github.com/keybase/client/blob/master/go/teams/teams.go#L180). But referenceImpName will have one.

If this does pass any tests that lookup with a conflict them I must be misunderstanding.

Reply:
this passes tests.  without this fix, the git team tests failed

Reply:
Ok. I don't understand how that can be but fine with me if it seems to work